src/status_im/ui/screens/chat/views.cljs

```diff
@@ -437,40 +438,46 @@
         (and
          mutual-contact-requests-enabled?
          one-to-one?
-         (not contact-added?))]
+         (not contact-added?))
+        loading-messages? @(re-frame/subscribe [:chats/loading-messages? chat-id])
+        all-loaded? @(re-frame/subscribe [:chats/all-loaded? chat-id])]
+
+    (if (or loading-messages? (not chat-id) (not all-loaded?))
```

Comment:
Also, I am not seeing any new-ui flag, wouldn't this will also affect old ui?

Reply:
no, because this entire view for new-ui only